geometry_msgs/Point center
float32 width
float32 height
float32 objectness
float32[] class_probability
